crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -312,7 +312,13 @@ where
     T: Bundle + 'static,
 {
     fn write(self, world: &mut World) {
-        world.entity_mut(self.entity).insert_bundle(self.bundle);
+        if let Some(mut entity) = world.get_entity_mut(self.entity) {
+            entity.insert_bundle(self.bundle);
+        } else {
+            panic!("Could not insert a bundle for entity {:?} because it no longer exists.\n\
```

Comment:
"no longer" implies that it did exist at some point, which isn't necessarily true (although this should generally be true)

Reply:
Perhaps? I think that making error messages 'worse' because someone has manually created an entity id is not the tradeoff I'd prefer to make.

We can change this to "doesn't exist" though if you're adamant.

Reply:
Acted on

Reply:
Fair fair:
![image](https://user-images.githubusercontent.com/2694663/131742964-124e3b99-3055-4cd0-bd9f-a430d6a3172d.png)